SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -221,11 +221,19 @@ void IPreInitilizeLowLevelSystem.SetCallback(Action<LowLevelSystem, SystemCallba
         #endregion
 
         #region init/close
+        [Obsolete("Not sure why this method is in here, the system should already be initialized when passed in through the constructor")]
         public void Initialize(int maxchannels, LowLevelInitFlags flags)
```

Comment:
There's no requirement to initialize the low level system before passing it into the constructor. Indeed the only way to do that would be to interact directly with the low level FMOD wrapper which we want to avoid as much as possible!

Additionally all the stuff in [IPreInitializeLowLevelSystem.cs](https://github.com/martindevans/SupersonicSound/blob/master/SupersonicSound/LowLevel/IPreInitilizeLowLevelSystem.cs) has to be done *before* the system is initialized (as the name implies).